cmd/fluent-manager/fluentbit/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Build the manager binary
+FROM golang:1.16 as builder
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY cmd/fluent-manager/fluentbit/main.go main.go
+COPY api api/
+COPY controllers controllers/
+COPY pkg pkg/
+
+# Build
+RUN CGO_ENABLED=0 GO111MODULE=on go build -a -o manager main.go
+
+# Use distroless as minimal base image to package the manager binary
+# Refer to https://github.com/GoogleContainerTools/distroless for more details
+FROM gcr.io/distroless/static:nonroot
```

Comment:
We'd better sync gcr distroless to https://hub.docker.com/repository/docker/kubesphere/distroless-static and use that instead

Reply:
Let's replace it when the dockerhub address supports both arm/amd arches in one manifest. Because we use buildx to push image.

Reply:
OK.